strongbox-user-management/src/main/java/org/carlspring/strongbox/authentication/registry/support/ExternalAuthenticatorsHelper.java

```diff
@@ -17,20 +17,20 @@
 /**
  * @author Przemyslaw Fusik
  */
-public class AuthenticatorsClassLoader
+public class ExternalAuthenticatorsHelper
```

Comment:
@sbespalov / @fuss86 ,

After this proposed fix, I believe we can proceed with merging the pull, right?